src/tests/grid/test_grids.cc

```diff
@@ -453,21 +456,25 @@ CASE( "test_cubedsphere" ) {
             grid->projection().lonlat2xy( crd );
             pointXYs_from_LonLat.push_back( crd );
         }
-        EXPECT( pointLonLats.size() == 6 * resolution * resolution + 2 );
-        EXPECT( pointXYs.size() == 6 * resolution * resolution + 2 );
-        EXPECT( grid.size() == 6 * resolution * resolution + 2 );
+        int numAdditionalPoints = 0;
+        if (s.substr(s.rfind("-")-1, 1) == "L") {
+            numAdditionalPoints = 2;
+        }
+        EXPECT( pointLonLats.size() == 6 * resolution * resolution + numAdditionalPoints );
+        EXPECT( pointXYs.size() == 6 * resolution * resolution + numAdditionalPoints );
+        EXPECT( grid.size() == 6 * resolution * resolution + numAdditionalPoints );
 
         // Note that with nodal points on the cubed-sphere
         // for a equiangular and equidistant projections and a resolution of 2 are the same.
         if ( resolution == 2 ) {
             constexpr double rpi     = M_PI;
             constexpr double rad2deg = 180. / rpi;
             double cornerLat         = rad2deg * std::atan( std::sin( rpi / 4.0 ) );
-            double tolerance         = 1e-13;
+            double tolerance         = 1e-4;
```

Comment:
If this tolerance value is too large I can enter more digits for the cell-center `expectedLatLon` values, but this is as small as it can be with the current expected values.